oeps/processes/oep-0058-proc-translations-management.rst

```diff
@@ -0,0 +1,169 @@
+OEP-58: Using the GitHub Transifex App and a Separate Translation Repo as a Replacement for the edx-transifex-bot
+#################################################################################################################
+
+.. list-table::
+
+   * - OEP
+     - :doc:`OEP-58 <oep-0058-proc-translations-management>`
+   * - Title
+     - Using the GitHub Transifex App and a Separate Translation Repo as a Replacement for the edx-transifex-bot
+   * - Last Modified
+     - 2022-08-08
+   * - Authors
+     - 
+       * Carlos Muniz <cmuniz@tcril.org>
+       * Feanil Patel <feanil@tcril.org>
+       * Sarina Canelake <sarina@tcril.org>
+   * - Arbiter
+     - Ned Batchelder <ned@edx.org>
+   * - Status
+     - Accepted
```

Comment:
Status should not be Accepted yet. Probably "Draft" or "Under Review"

Reply:
Changing it to "Under Review"